src/ui/components.tsx

```diff
@@ -46,13 +46,16 @@ export const CheckMkAsyncSelect = function <Key extends keyof RequestSpec, Value
   const [counter, setCounter] = React.useState(0);
   const [autocompleteError, setAutocompleteError] = React.useState('');
 
-  function findValueInOptions() {
-    const result = options.find((opt) => opt.value === value);
-    if (result) {
-      return result;
-    }
-    return null;
-  }
+  const findValueInOptions = React.useCallback(
```

Comment:
if we use value as a paramater to this function (which would make sense in my opinion) then we can move this function out of the react-function and remove the useCallback wrapper.